src/OrchardCore.Modules/OrchardCore.Alias/Indexes/AliasPartIndex.cs

```diff
@@ -1,13 +1,15 @@
-﻿using OrchardCore.ContentManagement;
+using OrchardCore.ContentManagement;
 using OrchardCore.Alias.Models;
 using YesSql.Indexes;
+using System;
 
 namespace OrchardCore.Alias.Indexes
 {
     public class AliasPartIndex : MapIndex
     {
         public string ContentItemId { get; set; }
         public string Alias { get; set; }
+        public bool Published { get; set; }
```

Comment:
you add this to the index but don't actually set it; or use it as far as i can see ?

Reply:
Also why not Latest AND Published fields. It won't change the number of records, but you can then also get explicitly the latest version whatever it is. Though there can be only one record in latest, and one record in published (they might both be true too, so one record).

Reply:
No, totally forgot to set it ;)

Added Latest as well